LegalInformation.txt

```diff
@@ -0,0 +1,14 @@
+Legal informations
+
+The openturns.org site are properties of :
+
+- Électricité de France, SA au capital social de 911 085 545 euros, immatriculée au RCS de paris sous le numéro 552 081 312, dont le siège social est situé 22-30 avenue de Wagram 75008 PARIS, FRANCE.
+- Airbus Group, SAS au capital social de 818 758 000 euros, immatriculée au RCS de Paris sous le numéro 341 535 094, dont le siège social est situé 37 Boulevard de Montmorency 75116 PARIS, FRANCE.
```

Comment:
not sure about it! 
I will say `Airbus SAS 2 Rond-Point Emile Dewoitine, 31700 Blagnac`

Reply:
Ok. Fixed.